src/evergreen/tst.py

```diff
@@ -17,6 +17,14 @@ def __init__(self, json, api):
         """Create an instance of a Test log."""
         super(Logs, self).__init__(json, api)
 
+    def stream(self):
```

Comment:
Out of curiosity, why is `class Tst` not spelled "Test" and this file is tst.py and not test.py?

Reply:
pytest does test discovery based on file and class names. So when it sees a file called `test.py` with a class named `Test` it thinks it is a test suite and attempts to run it. It gives a warning because it can't find any tests. Taking the 'e' out of the name means pytest doesn't get confused.